src/components/CardGrid/CardGrid.test.tsx

```diff
@@ -9,15 +10,17 @@ describe('<CardGrid>', () => {
   it('renders and matches snapshot', () => {
     const placeholderData = generatePlaylistPlaceholder();
     const { container } = render(
-      <CardGrid
-        playlist={placeholderData}
-        onCardHover={vi.fn()}
-        onCardClick={vi.fn()}
-        isLoading={false}
-        accessModel={'SVOD'}
-        isLoggedIn={true}
-        hasSubscription={true}
-      />,
+      <BrowserRouter>
```

Comment:
Why did you add this here? I don't think we need another BrowserRouter (in fact multiples might not play nicely together) and we already have 1 at the top level in App.tsx

Reply:
Oh this is a test file, my bad. You can use the wrapper functions in testUtils.tsx